citrination_client/client.py

```diff
@@ -55,6 +65,116 @@ def search(self, pif_query):
+        name will appear under the key "name", chemical formula under "chemical_formula", property name under
+        "property_name", value of the property under "property_value", units of the property under "property_units",
+        and reference DOI under "reference_doi".
+
+        This method is only meant for execution of very simple queries. More complex queries must use the search method
+        that accepts a :class:`.PifQuery` object.
+
+        :param name: One or more strings with the names of the chemical system to match.
+        :param chemical_formula:  One or more strings with the chemical formulas to match.
+        :param property_name: One or more strings with the names of the property to match.
+        :param property_value: One or more strings or numbers with the exact values to match.
+        :param property_min: A single string or number with the minimum value to match.
+        :param property_max: A single string or number with the maximum value to match.
+        :param property_units: One or more strings with the property units to match.
+        :param reference_doi: One or more strings with the DOI to match.
+        :param include_datasets: One or more integers with dataset IDs to match.
+        :param exclude_datasets: One or more integers with dataset IDs that must not match.
+        :param from_index: Index of the first record to match.
+        :param size: Total number of records to return.
+        :return: :class:`.PifSearchResult` object with the results of the query.
+        """
+        system_query = SystemQuery()
+
+        # Add the name operation
+        if isinstance(name, list):
+            system_query.names = [FieldOperation(extract_as='name', filter=Filter(equal=i)) for i in name]
+        elif isinstance(name, string_types):
+            system_query.names = [FieldOperation(extract_as='name', filter=Filter(equal=name))]
+        else:
+            system_query.names = [FieldOperation(extract_as='name')]
+
+        # Add the chemical formula operation
+        if isinstance(chemical_formula, list):
+            system_query.chemical_formula = \
+                [ChemicalFieldOperation(extract_as='chemical_formula', filter=ChemicalFilter(equal=i)) 
+                 for i in chemical_formula]
+        elif isinstance(chemical_formula, string_types):
+            system_query.chemical_formula = \
+                [ChemicalFieldOperation(extract_as='chemical_formula', filter=ChemicalFilter(equal=chemical_formula))]
+        else:
+            system_query.chemical_formula = \
+                [ChemicalFieldOperation(extract_as='chemical_formula')]
+        
+        # Generate the name query for the property
+        if isinstance(property_name, list):
+            property_name_query = \
+                [FieldOperation(extract_as='property_name', filter=Filter(equal=i)) for i in property_name]
```

Comment:
Both of those work just fine, actually. It looks like that page in the documentation is off. The property name can accept a list of FieldOperations (https://github.com/CitrineInformatics/python-citrination-client/blob/master/citrination_client/search/pif/query/core/property_query.py#L19). 

However, now that you mention it, even though both queries are allowed, the one that you wrote would be more efficient to execute! I am going to rewrite these blocks.